var/spack/repos/builtin/packages/sst-core/package.py

```diff
@@ -72,8 +89,14 @@ def configure_args(self):
         else:
             args.append("--disable-mpi")
 
+        if "+trackevents" in self.spec:
+            args.append("--enable-event-tracking")
+        if "+trackperf" in self.spec:
+            args.append("--enable-perf-tracking")
         if "+preview" in self.spec:
             args.append("--enable-preview-build")
+        if "+profile" in self.spec:
+            args.append("--enable-profile")
 
         args.append("--with-python=%s" % self.spec["python"].prefix)
         return args
```

Comment:
We also need to add the snipper of code to `sst-core`:
```
    def setup_run_environment(self, env):
        """Set-up runtime environment for SST Core."""

        env.prepend_path("LD_LIBRARY_PATH", self.spec["python"].prefix.lib)
```

See my last comment here: https://github.com/sstsimulator/sst-core/issues/948